src/main/java/com/github/manolo8/darkbot/config/NpcInfo.java

```diff
@@ -153,6 +154,10 @@ public static String getId(Enum<?> flag) {
         return lastSearchedId = className.concat(name);
     }
 
+    public String getFuzzyName(String originalName) {
```

Comment:
Needs to be exposed in the API so that LocatorPick's getFuzzyName can be compared to this without relying on internals. Also consider just having name be a field in NpcInfo so that the key (originalName) doesn't need to be passed in as param.